MAUI/ComboBox/Getting-Started.md

```diff
@@ -180,36 +205,45 @@ Now, populate this SocialMediaViewModel data in [ComboBox](https://help.syncfusi
 {% highlight C# %}
 
 SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
-comboBox.BindingContext = socialMediaViewModel;
-comboBox.ItemsSource = SocialMediaViewModel.SocialMedias;
+this.BindingContext = socialMediaViewModel;
+SfComboBox comboBox = new SfComboBox(); 
+comboBox.WidthRequest = 250;
+comboBox.HeightRequest = 50;
+comboBox.ItemsSource = socialMediaViewModel.SocialMedias;
+Content = comboBox;
 
 {% endhighlight %}
 {% endtabs %}
 
-N> Set the SocialMediaViewModel instance as the BindingContext of your control; this is done to bind properties of SocialMediaViewModel to `ComboBox`.
+N> Set the SocialMediaViewModel instance as the BindingContext of your control; this is done to bind properties of SocialMediaViewModel to ComboBox.
```

Comment:
Set the BindingContext of your page to an instance of SocialMediaViewModel. This allows you to bind the properties of SocialMediaViewModel to the Autocomplete control.

Reply:
Updated.